### PR TITLE
docs: added documentation for blog post

### PR DESCRIPTION
**Pull request by shariquerik**
Added documentation for Blog Post with feedback feature.
https://github.com/frappe/frappe/pull/13491

### CODE REVIEW

File: frappe_docs/www/docs/user/en/blog-post.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+add_breadcrumbs: 1
+title: Web Forms
+image: /assets/frappe_io/images/frappe-framework-logo-with-padding.png
```
Comment (gavindsouza) [inline frappe_docs/www/docs/user/en/blog-post.md:4]: You can get rid of this line

**Review by gavindsouza — COMMENTED** [review]
There seem to be inconsistencies in the screenshots. Can you make sure all screenshots are of the latest version?

File: frappe_docs/www/docs/user/en/blog-post.md (hunk 1 above)
Comment (shariquerik) [reply frappe_docs/www/docs/user/en/blog-post.md:4]: All pages have this title and description, that's why I kept it. I have changed the title and description now to Blog Post.

**Comment by shariquerik** [comment]
> There seem to be inconsistencies in the screenshots. Can you make sure all screenshots are of the latest version?

I updated the screenshots to the latest version.

**Review by gavindsouza — APPROVED** [review]

**Pull request merged**